packages/core/data/mm/world/beneath_the_well.yml

```diff
@@ -67,17 +67,15 @@
     "Beneath The Well Middle Section": "true"
     "Beneath The Well Sun Block": "has(MASK_GIBDO) && soul_redead_gibdo && has_milk"
   events:
-    RUPEES: "can_fight || has_weapon_range || has_explosives"
-    BUGS: "can_use_fire_arrows || (event(WELL_BIG_POE) && has_sticks)"
-  locations:
-    "Beneath The Well Skulltulla Chest": "has(MASK_GIBDO) && soul_redead_gibdo && has_bottle"
+    RUPEES: "(can_fight || has_weapon_range || has_explosives) && soul_wallmaster"
+    BUGS: "can_use_fire_arrows || (has_sticks && has_big_poe && has(MASK_GIBDO) && soul_redead_gibdo)"
```

Comment:
not sure why you put a Gibdo requirement here since it's trivial to just grab the bug. (should probably require bottle though)

Reply:
The bugs are behind a web here and there's no pre-lit torch in this room.  Without Fire Arrows, you would need to light a stick in the previous room and carry it into this one, which would require giving a Big Poe to the Gibdo.

Will add bottle requirement to this.

The bugs also can't be reached without getting past a Skulltula, which either requires combat or shield, so adding that as a requirement as well.

Reply:
why shield?

Reply:
Shield lets you backflip past the Skulltula without killing it, which you can't do without the shield (or perhaps you can, but if you can it's non-trivial). Perhaps this would fall under the recoil flip umbrella though and we wouldn't want to include it? Will put video on Discord for reference.

Reply:
Logic to kill / get past the Skulltula removed. Being able to burn the web implies that you are able to get kill the Skulltula as well.